docs/community/insight-pages.md

```diff
@@ -0,0 +1,12 @@
+---
+id: insight-pages
+title: 'View your favorite projects in the Insight pages'
+sidebar_label: 'View your favorite projects in the Insight pages'
+keywords:
+- insight-pages
+---
+
+You know how you see the delicious looking images of pizza pies in the menu when you first walk in a pizza shop? Well, we wanted to replicate this feeling with our Insight pages. This feature is where you pick and choose the repositories that you often contribute to. It's a great way to stay updated on their growth.
```

Comment:
The tone is nice, but will be hard to translate the first sentence if we end up doing that in the future. Probably just go with the last sentence.